CHANGELOG.md

```diff
@@ -11,6 +11,106 @@ git log --first-parent
 
 Please visit our wiki [Changelog](https://github.com/ginkgo-project/ginkgo/wiki/Changelog) for unreleased changes.
 
+## Version 1.7.0
+
+The Ginkgo team is proud to announce the new Ginkgo minor release 1.7.0. This release brings new features such as:
+- Complete GPU-resident sparse direct solvers feature set and interfaces,
+- Sparse batched iterative kernels support for Dense and ELL matrix types and the BiCGSTAB solver,
+- A new MC64 reordering,
+- MPI support for the SYCL backend,
+- Improved Cholesky factorization performance,
+- Improved ParILU(T)/ParIC(T) preconditioner convergence,
+and more!
+
+If you face an issue, please first check our [known issues page](https://github.com/ginkgo-project/ginkgo/wiki/Known-Issues) and the [open issues list](https://github.com/ginkgo-project/ginkgo/issues) and if you do not find a solution, feel free to [open a new issue](https://github.com/ginkgo-project/ginkgo/issues/new/choose) or ask a question using the [github discussions](https://github.com/ginkgo-project/ginkgo/discussions).
+
+Supported systems and requirements:
++ For all platforms, CMake 3.16+
++ C++14 compliant compiler
++ Linux and macOS
+  + GCC: 5.5+
+  + clang: 3.9+
+  + Intel compiler: 2019+
+  + Apple Clang: 14.0 is tested. Earlier versions might also work.
+  + NVHPC: 22.7+
+  + Cray Compiler: 14.0.1+
+  + CUDA module: CMake 3.18+, and CUDA 10.0+ or NVHPC 22.7+
+  + HIP module: ROCm 4.5+
+  + DPC++ module: Intel OneAPI 2021.3+ with oneMKL and oneDPL. Set the CXX compiler to `dpcpp` or `icpx`.
+  + MPI: version 3.1+, ideally with GPUDirect support for best performance
++ Windows
+  + MinGW: GCC 5.5+
+  + Microsoft Visual Studio: VS 2019+
+  + CUDA module: CUDA 10.0+, Microsoft Visual Studio
+  + OpenMP module: MinGW.
+
+### Version support changes
+
++ CUDA 9.2 is no longer supported [#1382](https://github.com/ginkgo-project/ginkgo/pull/1382)
++ Ginkgo now requires CMake version 3.16 (and 3.18 for CUDA) [#1368](https://github.com/ginkgo-project/ginkgo/pull/1368)
+
+### Interface changes
+
++ `const` Factory parameters can no longer be modified through `with_*` functions, as this breaks const-correctness [#1336](https://github.com/ginkgo-project/ginkgo/pull/1336) [#1439](https://github.com/ginkgo-project/ginkgo/pull/1439)
+
+### Deprecations
+
++ The `device_reset` parameter of CUDA and HIP executors no longer has an effect, and its `allocation_mode` parameters have been deprecated in favor of the `Allocator` interface [#1315](https://github.com/ginkgo-project/ginkgo/pull/1315)
++ The CMake parameter `GINKGO_BUILD_DPCPP` has been deprecated in favor of `GINKGO_BUILD_SYCL`. [#1350](https://github.com/ginkgo-project/ginkgo/pull/1350)
++ The `gko::reorder::Rcm` interface has been deprecated in favor of `gko::experimental::reorder::Rcm` based on `Permutation` [#1418](https://github.com/ginkgo-project/ginkgo/pull/1418)
+
```

Comment:
Should we also list other deprecations such as 
1. `gko::lend` 
2. `*_factory` is deprecated 
3. `permute_mask` not supported anymore
4. `RelativeResidualNorm` and `AbsoluteResidualNorm` being deprecated
5. `ConstArrayView` deprecated
6. `gko::solver::set_complex_subpsace()` deprecated
7. `MachineTopology` deprecated

Reply:
We should maybe have a deprecation summary page on our wiki instead or something like this, as the list is growing large. I am also testing if Doxygen can generate a nice summary list instead, that would be ideal.